website/source/docs/providers/aws/r/route.html.markdown

```diff
@@ -35,12 +35,14 @@ The following arguments are supported:
 * `destination_cidr_block` - (Required) The destination CIDR block.
 * `vpc_peering_connection_id` - (Optional) An ID of a VPC peering connection.
 * `gateway_id` - (Optional) An ID of a VPC internet gateway or a virtual private gateway.
```

Comment:
Gotcha. I kept digging and found the [struct definition](https://github.com/aws/aws-sdk-go/blob/18177cad2597e3cac4173b4f267b784abdb3c854/service/ec2/api.go#L9097) in the Go SDK, which clarified it for me too.

I wonder why they did that. Anyways, I understand what you're saying now. Thanks for the patience.